CHANGELOG.md

```diff
@@ -12,13 +13,17 @@
 * Ensure nginx restarts on frontends after lua-related changes
 * Updated nginx's logrotate config with proper log ownership.
 
+### oc\_erchef 1.2.0
+* add basic multikey/key rotation support. THis is not yet exposed via
```

Comment:
Small typo: THis -> This

Also, you probably want to say "an API" or "the API" rather than just API.